.github/workflows/config/spelling_allowlist.txt

```diff
@@ -236,19 +254,21 @@ preprocessor
 probability
 programmatically
 pybind
+pyscf
 quantize
 quantized
 qubit
 qubits
 qudit
 qudits
-reStructuredText
 reconfigurable
+reStructuredText
 runtime
 runtimes
 rvalue
 scalability
 scalable
+scf
```

Comment:
I don't think this is a word and therefore should probably not be added to the allowlist.